packages/terra-date-input/src/DateInputUtil.js

```diff
@@ -238,6 +238,21 @@ class DateInputUtil {
         : `(${props.intl.formatMessage({ id: 'Terra.date.input.dayFormatLabel' })} ${props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${props.intl.formatMessage({ id: 'Terra.date.input.yearFormatLabel' })})`
     );
   }
+
+  static getAriaDescriptionId(props, formatDescriptionId, inputAttributes) {
```

Comment:
taking a page out of @mjhenkes book, can we update this function to accept an object? this will give context on what each parameter is, each time we call the function.
```suggestion
  static getAriaDescriptionId(options) {
  const { props, id, inputAttributes } = options;
```

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1263/commits/c5635d80e012c9ab0b3a17286f1672eba2bcb8e8